### DISCUSSION TITLE
v2.0.8

### DISCUSSION THREAD

**Discussion by aaronhelton** [dag-hammarskjold-library/dlx-rest#1179] [General]
## What's Changed

User-testable, Editor: 
* Constrain auth 375 to "Mr.", "Ms.", and "Unknown" by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1144
* Add 491 and 591 to subject, agenda, agenda_subject, and agenda_title by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1131
* Update build-logical-fields to ensure proper logical field assignments, closing #1018  by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1164
* Add 700$g to auth results display by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1170
* restore add saved records to basket by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1160
* Implement auth 905 save action by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1168
* Fix wildcard searches (e.g., 999:*) by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1166
* update logical fields to include 711 in country_org by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1177

File Upload
* fix issues with back button and new file selection on file uploads page by @Yalshire2004 in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1136
* Export upload results in csv by @Yalshire2004 in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1137
* Update intro text on File Upload page by @Yalshire2004 in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1153
* Add user as additional column in file management UI by @Yalshire2004 in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1157
* Use underscores only to replace slashes in document symbols when naming files by @Yalshire2004 in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1158
* Limit file upload size to a total of 1 GB by @Yalshire2004 in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1176

Non-user-testable:
* Bump cryptography from 39.0.1 to 41.0.0 by @dependabot in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1122
* re-fix the ssl issue that was supposed to be fixed last time by @aaronhelton in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1129
* save upload results in mongodb by @Yalshire2004 in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1142
* support atlas production by @aaronhelton in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1149
* Bump cryptography from 41.0.0 to 41.0.2 by @dependabot in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1163
* Bump certifi from 2022.12.7 to 2023.7.22 by @dependabot in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1175

**Full Changelog**: https://github.com/dag-hammarskjold-library/dlx-rest/compare/v2.0.7...v2.0.8

<hr /><em>This discussion was created from the release <a href='https://github.com/dag-hammarskjold-library/dlx-rest/releases/tag/v2.0.8'>v2.0.8</a>.</em>

**Comment by aaronhelton** [comment]
This will be deployed to QAT ASAP, but requires that build-logical-fields be run, which takes time.

**Comment by viola-v** [comment]
Tested in QAT (1 Aug 2023): 
Constrain auth 375 to "Mr.", "Ms.", and "Unknown" by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1144 **=> works as intended**
Add 491 and 591 to subject, agenda, agenda_subject, and agenda_title by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1131 **=> works as intended**
Update build-logical-fields to ensure proper logical field assignments, closing https://github.com/dag-hammarskjold-library/dlx-rest/issues/1018 by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1164 **=> works as intended**
Add 700$g to auth results display by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1170 **=> works as intended**
restore add saved records to basket by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1160 **=> works as intended**
Implement auth 905 save action by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1168  => checking with Geneva if functionality works for them
Fix wildcard searches (e.g., 999:*) by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1166 **=> works as intended**
update logical fields to include 711 in country_org by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1177 **=> works as intended**
